tendermint/src/types/request/init_chain.rs

```diff
@@ -3,19 +3,13 @@ use crate::types::{
     time::Timestamp,
 };
 
-#[derive(Clone, PartialEq, Eq, ::prost::Message, serde::Serialize, serde::Deserialize)]
```

Comment:
Be aware that `prost::Message` implements `Debug` too

Reply:
Yes, we can add Debug derive if/when necessary.